templates/index

```diff
@@ -276,9 +276,10 @@ cisco_nxos_show_license_usage.textfsm, .*, cisco_nxos, sh[[ow]] lic[[ense]] us[[
 cisco_nxos_show_processes_cpu.textfsm, .*, cisco_nxos, sh[[ow]] proc[[esses]] c[[pu]]
 cisco_nxos_show_vrf_interface.textfsm, .*, cisco_nxos, sh[[ow]] vrf int[[erface]]
 cisco_nxos_show_access-lists.textfsm, .*, cisco_nxos, sh[[ow]] acc[[ess-lists]]
-cisco_nxos_show_environments.textfsm, .*, cisco_nxos, sh[[ow]] envi[[ronments]]
```

Comment:
@diepes After digging into this a bit more during the review, we don't believe `show environments` is a valid command so we should be able to actually remove that and just keep the following line (notice the edit as well).

`cisco_nxos_show_environment.textfsm, .*, cisco_nxos, sh[[ow]] env[[ironment]]`

Reply:
Do you mind updating the new line with the above so users can use `show env` command?

Reply:
Fixed